Hermes/appscale/hermes/producers/node_stats.py

```diff
@@ -70,6 +70,7 @@ class NodePartition(object):
 class NodeNetwork(object):
   bytes_sent = attr.ib()
   bytes_recv = attr.ib()
+  bytes_recv = attr.ib()
```

Comment:
I don't understand this line.

Reply:
huh... I don't either I think that was an accident